File: src/main/kotlin/net/merayen/elastic/uinodes/list/wave_1/Info.java
package net.merayen.elastic.uinodes.list.wave_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override
	public String getName() {
		return "Signal generator";
	}

	@Override
	public String getDescription() {
		return "Generates audio signal";
	}

	@Override
	public String[] getCategories() {
		return new String[]{"Audio"};
	}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/autocorrelation_1/Info.kt
package net.merayen.elastic.uinodes.list.autocorrelation_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "Auto-correlation"
	override fun getDescription() = "Figures out distance between two audio signals by looking for patterns"
	override fun getCategories() = arrayOf("Audio")
}